x-pack/plugins/security_solution/public/resolver/view/submenu.tsx

```diff
@@ -74,21 +81,44 @@ const OptionList = React.memo(
                 };
           })
     );
-    return useMemo(
-      () => (
-        <EuiSelectable
-          singleSelection={true}
-          options={options}
-          onChange={(newOptions) => {
-            setOptions(newOptions);
-          }}
-          listProps={{ showIcons: true, bordered: true }}
-          isLoading={isLoading}
-        >
-          {(list) => <OptionListItem>{list}</OptionListItem>}
-        </EuiSelectable>
-      ),
-      [isLoading, options]
+
+    const actionsByLabel: Record<string, () => unknown> = useMemo(() => {
```

Comment:
future refactoring notes: `actionsByLabel` is only used in the below `useMemo` (`selectableProps`) so perhaps we can inline this there.